novawallet/Modules/NetworkManageNode/NetworkManageNodeViewController.swift

```diff
@@ -0,0 +1,111 @@
+import UIKit
+import SoraFoundation
+
+final class NetworkManageNodeViewController: UIViewController, ViewHolder {
```

Comment:
I thought we could use ```ModalPickerViewController``` for that bottomsheet. Probably we can leave it as it is for now.

Reply:
oh, haven't seen this one